client/source/js/modules/model/cost-coverage.html

```diff
@@ -22,34 +22,13 @@
           class="txbox"
           ng-options="p as p.name group by p.category for p in programs"
           ng-change="changeProgram()"
+          ng-click="generateCurves()"
```

Comment:
@cliffckerr what was you intention here?

Right now this would draw the existing program. Did you want to automatically run the program once switched?